DataFormats/CTPPSReco/interface/CTPPSLocalTrackLite.h

```diff
@@ -11,75 +11,147 @@
 
     /// returns the horizontal track position uncertainty
-    float getXUnc() const
+    inline float getXUnc() const
     {
       return x_unc;
     }
 
     /// returns the vertical track position
-    float getY() const
+    inline float getY() const
     {
       return y;
     }
 
     /// returns the vertical track position uncertainty
-    float getYUnc() const
+    inline float getYUnc() const
     {
       return y_unc;
     }
 
     /// returns the track time
-    float getTime() const
+    inline float getTime() const
     {
       return time;
     }
 
     /// returns the track time uncertainty
-    float getTimeUnc() const
+    inline float getTimeUnc() const
     {
       return time_unc;
     }
 
+    /// returns the track horizontal angle
+    inline float getTx() const
+    {
+        return tx;
+    }
+
+    /// returns the track horizontal angle uncertainty
+    inline float getTxUnc() const
+    {
+        return tx_unc;
+    }
+    
+    /// returns the track vertical angle
+    inline float getTy() const
+    {
+        return ty;
+    }
+    
+    /// returns the track vertical angle uncertainty
+    inline float getTyUnc() const
+    {
+        return ty_unc;
+    }
+    
+    /// returns the track fit chi Squared over NDF
+    inline float getChiSquaredOverNDF() const
+    {
+        return chiSquaredOverNDF;
+    }
+
+    /// returns the track reconstruction info byte
+    inline unsigned int getRecoInfo() const
+    {
+        return reco_info;
+    }
+
+    /// returns the number of points used for fit
+    inline unsigned short getNumberOfPointsUsedForFit() const
+    {
+        return numberOfPointsUsedForFit;
+    }   
+
   protected:
     /// RP id
     uint32_t rpId;
 
+    /// local track parameterization
+    /// x = x0 + tx*(z-z0), y = y0 + ty*(z-z0)
+    /// x0, y0, z-z0 in mm
+    /// z0: position of the reference scoring plane (in the middle of the RP)
+
     /// horizontal hit position and uncertainty, mm
     float x, x_unc;
 
     /// vertical hit position and uncertainty, mm
     float y, y_unc;
 
+    /// horizontal angle and uncertainty, x = x0 + tx*(z-z0)
+    float tx, tx_unc;
+
+    /// vertical angle and uncertainty, y = y0 + ty*(z-z0)
+    float ty, ty_unc;
+
+    /// fit chi^2 over NDF
+    float chiSquaredOverNDF;
+
+    /// Track information byte for bx-shifted runs: 
+    /// reco_info = notShiftedRun    -> Default value for tracks reconstructed in non-bx-shifted ROCs
+    /// reco_info = allShiftedPlanes -> Track reconstructed in a bx-shifted ROC with bx-shifted planes only
+    /// reco_info = noShiftedPlanes  -> Track reconstructed in a bx-shifted ROC with non-bx-shifted planes only
+    /// reco_info = mixedPlanes      -> Track reconstructed in a bx-shifted ROC both with bx-shifted and non-bx-shifted planes
+    /// reco_info = invalid          -> Dummy value. Assigned when reco_info is not computed (i.e. non-pixel tracks)
+    CTPPSPixelLocalTrack::reconstructionInfo reco_info;
```

Comment:
I named it reco_info to use the match it with the data member of CTPPSPixelLocalTrack, but it actually makes more sense if the name includes "Pixel" at this point, since it is meaningful only for pixels.
Changing it

Reply:
I still want to avoid the unnecessary header inter-dependency of CTPPSLocalTrackLite.h  using CTPPSPixelLocalTrack.h.
So the "common" place noted in https://github.com/cms-sw/cmssw/pull/25575#discussion_r246449879 can instead become pixel-specific and appear as CTPPSPixelLocalTrackRecoInfo.h